@navikt/core/react/src/alert/local-alert/root/LocalAlertRoot.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { forwardRef } from "react";
+import { BaseAlert } from "../../base-alert";
+import {
+  LocalAlertClose,
+  type LocalAlertCloseProps,
+} from "../close/LocalAlertClose";
+import {
+  LocalAlertContent,
+  type LocalAlertContentProps,
+} from "../content/LocalAlertContent";
+import {
+  LocalAlertHeader,
+  type LocalAlertHeaderProps,
+} from "../header/LocalAlertHeader";
+import {
+  LocalAlertTitle,
+  type LocalAlertTitleProps,
+} from "../title/LocalAlertTitle";
+
+interface LocalAlertProps
+  extends Omit<
+    BaseAlert.RootProps,
+    "type" | "global" | "statusType" | "data-color"
+  > {
+  variant: BaseAlert.RootProps["variant"];
+}
```

Comment:
`undefined` is still allowed.

```suggestion
interface LocalAlertProps
  extends Omit<
    BaseAlert.RootProps,
    "type" | "global" | "statusType" | "data-color"
  > {
  variant: Exclude<BaseAlert.RootProps["variant"], undefined>;
}
```

```suggestion
type LocalAlertProps = Omit<
  BaseAlert.RootProps,
  "type" | "global" | "statusType" | "data-color"
> &
  Required<Pick<BaseAlert.RootProps, "variant">>;
```